Session2.Rmd

```diff
@@ -44,18 +44,23 @@ The `st_geometry_type()` function gives us information about the geometry type,
 st_geometry_type(boundary_Delft)
 ```
 
-The `st_crs()` function gives us the coordinate reference system used by the shapefile, which in this case is `WGS84`. This is not what we want, so we will use the `st_transform()` function to reproject the file to the right CRS. For the `crs` argument we can use the EPSG code of the CRS we want to use. For the Netherlands, we want to use the Amersfort / RD New projection, which has the EPSG code 28992.
+The `st_crs()` function gives us the coordinate reference system used by the shapefile, which in this case is `WGS84`, which has the unique reference code `EPSG: 4326`. The `st_bbox()` function shows the extent of the layer. As `WGS84` is a **geographic CRS**, the extent of the shapefile is displayed in degrees.
 
 ```{r}
 st_crs(boundary_Delft)
 st_bbox(boundary_Delft)
 ```
+
+We need a **projected CRS**, which in the case of the Netherlands is typically the Amersfort / RD New projection. To reproject our shapefile, we will use the `st_transform()` function. For the `crs` argument we can use the EPSG code of the CRS we want to use, which is 28992 for the Amersfort / RD New projection. Notice that the bounding box is measured in meters after the transformation.
```

Comment:
Depending on who the participants are you might want to give an example of what happens if you keep the CRS at WSG84 and why RD new is better suited for NL

Reply:
Okay, noted! I will keep this in mind (or please remind me if I forget). We do not really have time for a full explanation of projections, but we can link to some. I do show a comparison of the country in the two projections on a slide though, if I remember correctly.